docs/remote.md

```diff
@@ -0,0 +1,181 @@
+# Setting a Remote Environment <!-- omit in toc -->
+
+MLRun allows you to use write your code on a local machine while running your functions on a remote cluster. This tutorial explains how you can set this up.
+
+- [Prerequisites](#prerequisites)
+- [Configure Remote Environment](#configure-remote-environment)
+  - [Set Environment Variables](#set-environment-variables)
+- [IDE Configuration](#ide-configuration)
+- [Remote Environment from PyCharm](#remote-environment-from-pycharm)
+- [Remote Environment from VSCode](#remote-environment-from-vscode)
+  - [Create Environment File](#create-environment-file)
+  - [Create Python Debug Configuration](#create-python-debug-configuration)
+  - [Set Environment File in Debug Configuration](#set-environment-file-in-debug-configuration)
+- [Set Environment Variables in a Terminal](#set-environment-variables-in-a-terminal)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Before you begin, ensure that the following prerequisites are met:
+
+1. Install MLRun locally.
+
+    You need to install MLRun locally and make sure the that the MLRun version you install is the same as the MLRun service version. Install a specific version using the following command; replace the `<version>`  placeholder with the MLRun version number (e.g., `0.5.4`):
+ 
+    ```sh
+    pip install mlrun==<version>
+    ```
+
+    If you already installed a previous version of MLRun you should first uninstall it by running:
+
+    ```sh
+    pip uninstall -y mlrun
+    ```
+
+2. Ensure that you have remote access to your MLRun service (i.e., to the service's NodePort on the remote Kubernetes cluster).
+
+## Configure Remote Environment
+
+### Set Environment Variables
+
+Set environment variables to define your MLRun configuration. As a minimum requirement &mdash;
+
+1. Set `MLRUN_DBPATH` to the URL of the remote MLRun database/API ervice; replace the `<...>` placeholders to identify your remote target:
+
+    ```ini
+    MLRUN_DBPATH=<API endpoint of the MLRun APIs service engpoint; e.g., "https://mlrun-api.default-tenant.app.mycluster.iguazio.com">
+    ```
+    
+2. In order to store the artifacts on the remote server, you need to set the `MLRUN_ARTIFACT_PATH` to the desired root folder of your artifact. You can use `{{run.project}}` to include the project name in the path `{{run.uid}}` to include the specific run uid in the artifact path. For example:
+
+    ```ini
+    MLRUN_ARTIFACT_PATH=/User/artifacts/{{run.project}}
+    ```
+
+3. If the remote service is on an instance of the Iguazio Data Science Platform (**"the platform"**), set the following environment variables as well; replace the `<...>` placeholders with the information for your specific platform cluster:
+
+    ```ini
+    V3IO_USERNAME=<username of a platform user with access to the MLRun service>
+    V3IO_API=<API endpoint of the web-APIs service endpoint; e.g., "https://webapi.default-tenant.app.mycluster.iguazio.com">
```

Comment:
Please verify this with @yaronha I think this might need to be the webAPI's `HTTPS_DIRECT` URL, e.g. `https://default-tenant.app.hedingber-210-1.iguazio-cd1.com:8444`

Reply:
changed to direct